diff --git a/dist/build.js b/dist/build.js
index 29144bc5cdb118655229a1f8998c3168170ca257..ba0595bd0a2a9789aac96ba8e7c97b037da34536 100644
--- a/dist/build.js
+++ b/dist/build.js
@@ -20,10 +20,12 @@ function validateTargets(targets) {
         if (!ext.startsWith(".")) {
             throw new Error(`targets[${i}].extname must be started with ".".`);
         }
-        const existedIndex = extMap.get(ext);
-        if (existedIndex != null) {
-            throw new Error(`targets[${i}].extname is already used in targets[${existedIndex}].extname`);
-        }
+        // PATCHED by Splitgraph
+        // NOTE: duplicate extensions are okay as long as targets have different outDir and tsBuildInfoFile
+        // const existedIndex = extMap.get(ext);
+        // if (existedIndex != null) {
+        //     throw new Error(`targets[${i}].extname is already used in targets[${existedIndex}].extname`);
+        // }
         extMap.set(ext, i);
     }
 }
